docs/maintainers/maintainers-guide.md

```diff
@@ -0,0 +1,152 @@
+---
+id: maintainers-guide-to-open-sauced
+title: 'Maintainers Guide to OpenSauced'
```

Comment:
The titles in some files accross the repo have single quotes, others have double quotes, and several have no quote.
Which one will we use for consistency?